cmd/client/lib/cli.go

```diff
@@ -209,3 +234,18 @@ func buildClientHost(clientListenAddr string, relayMultiaddr []ma.Multiaddr) (*p
 	}
 	return ps, nil
 }
+
+// chainInfoFromGroupTOML reads a drand group TOML file and returns the chain info.
+func chainInfoFromGroupTOML(path string) (*chain.Info, error) {
+	gt := &key.GroupTOML{}
+	_, err := toml.DecodeFile(path, gt)
```

Comment:
can we - if this fails - also try to decode it from JSON?

it would be nice to curl the /info endpoint and be able to use that as the group info to bootstrap :)